docs/tutorials/access-service-across-clusters.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Access service acrosss clusters within native service name
+---
+
+In Karmada, the MultiClusterService can enable users to access services across clusters with the native service name, like `foo.svc`, with the aim of providing a seamless experience for users to access services across clusters.
```

Comment:
Is foo.svc the name of the service or the domain to access?

Reply:
should be domain name